packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -119,7 +122,7 @@ const AppContents = observer(({ children }) => {
                 'app-contents--is-scrollable': is_cfd_page || is_cashier_visible,
                 'app-contents--is-hidden': platforms[platform],
                 'app-contents--is-onboarding': window.location.pathname === routes.onboarding,
-                'app-contents--is-dtrader-v2': dtrader_v2_enabled,
+                'app-contents--is-dtrader-v2': dtrader_v2_enabled_mobile || dtrader_v2_enabled_desktop,
```

Comment:
It looks like `app-contents--is-dtrader-v2` is specifically for mobile or tablet screen sizes. Do we really need the || condition here?

Reply:
we dont have a design for v2 desktop yet so i didnt create another classname. i just keep it in this layout for completeness and make sure its not missed.

When we begin we'll have to manage this.